lib/CL/devices/level0/pocl-level0.cc

```diff
@@ -678,6 +819,13 @@ int pocl_level0_link_program(cl_program Program, cl_uint DeviceI,
                              int CreateLibrary) {
   cl_device_id Dev = Program->devices[DeviceI];
   Level0Device *Device = (Level0Device *)Dev->data;
+
+  if (Dev->compiler_available == CL_FALSE ||
```

Comment:
I think we can assume ! works for CL_FALSE, let's use that, more readable.

Reply:
Done.